src/main/resources/application.properties

```diff
@@ -8,8 +8,8 @@ server.port: 8090
 rules.path: /ArtifactRules_new.json
 
 logging.level.root: INFO
-logging.level.org.springframework.web: INFO
-logging.level.com.ericsson.ei: INFO
+logging.level.org.springframework.web: DEBUG
+logging.level.com.ericsson.ei: DEBUG
```

Comment:
I think it can cause fail of travis build because a huge number of logs

Reply:
This is temporary. I hope it will give me more information to debug the current issue in Travis.